dags/hindawi/hindawi_api_client.py

```diff
@@ -29,8 +28,33 @@ def get_articles_metadata(self, parameters, doi=None):
                 break
             return response_content
 
-    # TODO: don't forget to download pdf when it will be needed
+    def get_file(self, parameters, extension, doi=None):
+        path_segments_for_files = ["journals", "ahep"]
+        try:
+            doi_part = doi.replace("10.1155/", "") + f".{extension}"
```

Comment:
@ErnestaP could we double check with Anne why we need that?

Reply:
Still waiting for reply

Reply:
She said yes, it is for all of articles' pdfs, xmls, pdfas